schema/config-schema.json

```diff
@@ -144,6 +144,15 @@
                 "selinuxLabel": {
                     "type": "string"
                 },
+                "ioPriority": {
+                    "class": "string",
+                    "enum": [
+                        "IOPRIO_CLASS_RT",
+                        "IOPRIO_CLASS_BE",
+                        "IOPRIO_CLASS_IDLE"
+                    ],
+                    "priority": "integer"
+                },
```

Comment:
I think the enum is defined in the wrong location, because this `enum` would now apply to `ioPriority` as a whole, but the enum defines values for `class` only.

So `class` should probably be defined as `"#/definitions/IOPriorityClass"` (e.g.) instead of a `"string"`. Similar to 

https://github.com/opencontainers/runtime-spec/blob/720792f25ae6e9ee6b1332db698f37659e69ce8d/schema/defs-linux.json#L102

https://github.com/opencontainers/runtime-spec/blob/504f70ef81a4f5bfb337e5a694c9552e22bf9935/schema/defs-linux.json#L77-L88

Reply:
Oh! I think https://github.com/opencontainers/runtime-spec/pull/1206 was meant to address that, correct @giuseppe ?